src/pages/home/sidebar/SidebarLinksData.js

```diff
@@ -76,7 +80,12 @@ function SidebarLinksData({isFocused, allReportActions, betas, chatReports, curr
         return reportIDs;
     }, [allReportActions, betas, chatReports, currentReportID, policies, priorityMode]);
 
-    const isLoading = SessionUtils.didUserLogInDuringSession() && isLoadingReportData;
+    useEffect(() => {
+        if (!isFirstTimeReportLoading) {
+            return;
+        }
+        setIsFirstTimeReportLoading(isLoadingReportData);
+    }, [isFirstTimeReportLoading, isLoadingReportData]);
```

Comment:
The code is not very intuitive about:

- Why do we initialize `isFirstTimeReportLoading` using `SessionUtils.didUserLogInDuringSession()`
- Why do we need/use `isFirstTimeReportLoading` instead of `isLoadingReportData`

Would be good to add some comments, on why we are doing things like we are here.

I also have to get familiarized with `SessionUtils.didUserLogInDuringSession()` as I'm not what does this state represent and why does it matter for the LHN